packages/react-native-layout/.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = { 
+  extends: [
+    '../../.eslintrc.js',
+    '@bothrs/eslint-config/react',
+    '@bothrs/eslint-config/react-native',
+  ],
+  parserOptions: {
+    project: 'tsconfig.json',
+    tsconfigRootDir: __dirname,
```

Comment:
Q: Why is this necessary? Normally specifying the project field as `./tsconfig.json` is enough

Reply:
It isn't enough. The bug that I've showed before where either the formatting **or** the yarn lint is working still happened. This is due to the way that yarn workspaces works. When developing and changing files, everything assumes you start from the root. Once you run the yarn lint command on root however, it runs a yarn lint command from each package separately, which means their point of origin is different.

Adding the __dirname, fixes this by always working from the same origin point for both situations